docs/core/entity/todo.md

```
@@ -109,9 +109,9 @@ update.
 
 | Name        | Type             | Default      | Description                                                                                                                                     |
 | ----------- | ---------------- | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------- |
-| uid | <code>string &#124; None</code> | `None` | A unique identifier for the to-do item. This field is required for updates and the entity state.
-| summary | <code>string &#124; None</code>  | `None` | A title or summary of the to-do item. This field is required for the entity state.
-| status | <code>TodoItemStatus &#124; None</code> | `None` | Defines the overall status for the to-do item, either `NEEDS_ACTION` or `COMPLETE`. This field is required for the entity state.
-| due | <code>datetime.date &#124; datetime.datetime &#124; None</code> | `None` | The date and time that a to-do is expected to be completed. As a datetime, must have a timezone.
-| description | <code>string &#124; None</code>  | `None` | A more complete description of the to-do item than that provided by the summary.
-| completed | <code>datetime.datetime &#124; None</code> | `None` | A timestamp indicating the time this item was last marked `COMPLETE`.
+| uid | `str \| None` | `None` | A unique identifier for the to-do item. This field is required for updates and the entity state.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)